docs/blog/2019-01-30-why-themes/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Why Themes?"
+date: "2019-01-30"
+author: "Kyle Mathews"
+tags: ["themes"]
+---
+
+> _A site as a function of your data_
+
+As some of you might have heard (or hoped), we're working on adding theme support to Gatsby. This blog posts is a short intro into how we're thinking about themes in Gatsby.
+
+This post is short on examples and concrete details. See [our other theme blog posts by Chris Biscardi](/blog/tags/themes) and his recent [talk at Gatsby Days on themes](https://youtu.be/wX84vXBpMR8).
+
+## Introduction & Goal
+
+Themes is a powerful new abstraction under development for Gatsby. They allow a developer to package up a site as a standalone product for others to use.
```

Comment:
```suggestion
Themes are a powerful new abstraction under development for Gatsby. They allow a developer to package up a site as a standalone product for others to use.
```

Reply:
I wasn't sure about this — I mean "themes" as the concept not a plurality of individual themes. /cc @shannonbux